actgen/agents/__init__.py

```diff
@@ -1,3 +1,5 @@
 from .randomagent import RandomAgent
-from .dqn import DQNAgent, QNet
-from .DirectedQNet import DirectedQNet
+from .dqn import DQNAgent
+from .directed_qnet import DirectedQNet
+from .action_dqn import ActionDQNAgent
+from .expert import PendulumExpert
```

Comment:
I go back and forth about whether it's good to include all these imports in the __init__.py

Reply:
It sometimes can lead to import warnings, but it's convenient if the modules mostly have the same name as the class they contain. If you notice import warnings, let me know, and we can resolve them. Otherwise let's leave it.

Reply:
How else could we import the modules without including them in `__init__.py`? 

I haven't seen any warning about this, except a warning about random seed:

> [W Context.cpp:69] Warning: torch.set_deterministic is in beta, and its design and  functionality may change in the future. (function operator())

Reply:
https://docs.python-guide.org/writing/structure/#packages

> A commonly seen issue is adding too much code to `__init__.py` files. When the project complexity grows, there may be sub-packages and sub-sub-packages in a deep directory structure. In this case, importing a single item from a sub-sub-package will require executing all `__init__.py` files met while traversing the tree.
> 
> Leaving an `__init__.py` file empty is considered normal and even good practice, if the package’s modules and sub-packages do not need to share any code.

The reason I sometimes like to add stuff to `__init__.py` is that it can simplify `import` statements and remove redundancy. Instead of `from agents.dqn import DQN` we can simply write `from agents import DQN`.